From 0d878da27e95a02483fb55642ce40e0f38096db1 Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Tue, 10 Apr 2018 17:12:41 +0200
Subject: [PATCH] Revert "GIT_SILENT: increase version"

This reverts commit 097f1c0aaafcbf03eb95b31583fb1ed713d9f810.

Leap 42.3 only has libical 1.0.1, but it still builds fine with this
lower version as well.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: kcalcore-19.08.1/CMakeLists.txt
===================================================================
--- kcalcore-19.08.1.orig/CMakeLists.txt	2019-08-25 15:02:23.000000000 +0200
+++ kcalcore-19.08.1/CMakeLists.txt	2019-09-06 08:27:56.794097979 +0200
@@ -39,7 +39,7 @@
 ########### Find packages ###########
 find_package(Qt5 ${QT_REQUIRED_VERSION} CONFIG REQUIRED Core Gui)
 
-set(LibIcal_MIN_VERSION "2.0")
+set(LibIcal_MIN_VERSION "1.0")
 find_package(LibIcal ${LibIcal_MIN_VERSION})
 set_package_properties(LibIcal PROPERTIES TYPE REQUIRED)
 if(LibIcal_VERSION VERSION_GREATER "2.98")
